### ISSUE TITLE
[Task]: Change sorting direction tx history

### ISSUE THREAD

**Issue by gjeee** [iotaledger/explorer#940]
### Task description

in the stardust (shimmer/iota) explorer the tx history for a specific address shows from old to new....if u push "load more" it adds more older records on top (out of visibility).....it would make more sense that the older records u want to see are immediately in "scope". so i would say change the sorting direction from new -> old.

### Requirements

n/a

### Acceptance criteria

n/a

### Creation checklist

- [ ] I have assigned this task to the correct people
- [ ] I have added the most appropriate labels
- [ ] I have linked the correct milestone and/or project

**Comment by begonaalvarezd** [comment]
@gjeee the tx history is from newest to order, what do you mean? Please share an example if you have an address that has the tx history reversed

**Comment by JannemanDev** [comment]
I agree dat the Load More button isn't really intuitive. You would expect the newly loaded (older) transactions to come into view _and_ on the bottom. Now it gets added on the top. Especially when the list is long and you are at the bottom and press the button screen will even scroll a bit to make it even more unclear whats happening and whats has been added.

Here's example (btw IOTA explorer suffers the same problem):
First screen:
![shimmer txs 1](https://github.com/iotaledger/explorer/assets/13236774/ee76cc44-f7ae-42ec-819d-1a905488d78c)

Second screen after Load more was pressed:
![shimmer txs 2](https://github.com/iotaledger/explorer/assets/13236774/41768274-e6c3-4b2f-8f49-6c6ef0c0396f)

**Comment by JannemanDev** [comment]
It should be top to bottom new to old and when you press the button the next batch of older txs should be added at the bottom. Maybe good idea to give these new batch a different background color so you know which ones were added by the last button press. 
Then we have two options:
-scroll the screen so you can press the load more button multiple times easily without moving you mouse (preferred)
-do not scroll the screen

**Comment by begonaalvarezd** [comment]
The tx history is being reworked, at the moment each row is an output (and the grouped color is because they are colored the same if they belong to the same tx), which isnt very intuitive. That is why you also have items added on top, the tx is the same, but there are more outputs added, which isnt great UX neither. Our UX improvements will hopefully fix all this https://github.com/iotaledger/explorer/pull/872

**Comment by begonaalvarezd** [comment]
Transaction history has been improved in stardust networks, please feel free to reopen if there is an issue 🌷

**Comment by gjeee** [comment]
wel not sure if your fix is already in production, but the tx ordering is now a complete mess. just check for yourself.

**Comment by begonaalvarezd** [comment]
> Transaction history has been improved in stardust networks, please feel free to reopen if there is an issue 🌷

what do you mean? the transaction history is now sorted from newest to oldest and grouped by transaction instead of outputs to improve usability

**Comment by begonaalvarezd** [comment]
> well does not look sorted to me....this is shimmer explorer:
> 
> ![image](https://private-user-images.githubusercontent.com/13155183/305752477-94fba041-d984-4590-b030-6278d5d6fed5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzMzU4NzcsIm5iZiI6MTcwODMzNTU3NywicGF0aCI6Ii8xMzE1NTE4My8zMDU3NTI0NzctOTRmYmEwNDEtZDk4NC00NTkwLWIwMzAtNjI3OGQ1ZDZmZWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDA5MzkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1YjBhODY5YTVhOWUxNzYyMDNkZjE5MzMyYjRhMjYzNmNiM2Q0NTFlMGRiZDE1NTdkNGY1OWI1NTFiYTNhMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qExGEce_6kBKjwK-ZIfwnT6dMKsGJbMEOvjyvNet55Y)

@gjeee Can you please share an address that has unsorted tx history? Its the first time I see it, all addresses I check are perfectly sorted

![image](https://github.com/iotaledger/explorer/assets/3624944/68f4a132-0c0c-4628-903b-94ea3d81a25d)
